action.yml

```diff
@@ -7,14 +7,17 @@ inputs:
     description: 'Input string that gets passed along to the docker build args'
     required: true
   tag_list:
-    description: 'List of tags that gets passed along to the docker build args'
+    description: 'List of comma-separated tags that gets passed along to the docker build args'
     required: true
```

Comment:
If we set `required: false`, this makes it optional, right? I don't want to force users to specify a tag list